MAINTAINERS.md

```diff
@@ -25,29 +99,57 @@ git push -u origin release-$now
 scripts/have-news HEAD^ > have-news.md
 ```
 
-Create a release PR, pasting `have-news.md` into the body.  Then build the SDK:
+- [ ] Create the release PR.
+
+The above should have pushed state to GitHub to let you create a
+new PR to merge the release branch back to the main branch. The PR
+name should follow the convention of previous releases, which is
+currently `chore(release): publish _release_label_`.  Paste
+`have-news.md` as the description of the PR.  Follow the example
+of previous releases for any other details of the PR.
+
+Creating this PR will also kick off the CI tests.
+
+- [ ] Build the NPM-installed SDK packages.
+
+While the above CI tests run, build the SDK:
 
 ```sh
 # Build all package generated files.
-yarn install
+yarn install --force
```

Comment:
I don't remember needing `--force`.. what did you run into that required it?

Also, I remember you were surprised that `yarn build` didn't compile the Go code.. did we conclude that that was ok? Or that we should also do `make -C packages/cosmic-swingset` to trigger the golang compile?

Reply:
The `--force` flag was needed to overcome one of the obstacles that I ran into during the release process. It could have been caused by a blunder that's now prevented by these more detailed instructions - but I'm not sure, and it was thought to be a good defensive measure for building the release.

I've created #7053 to discuss whether to integrate the golang components with yarn.